src/price_data.py

```diff
@@ -22,7 +22,7 @@
 import sqlite3
 import time
 from pathlib import Path
-from typing import Any, Optional, Union
+from typing import Any, Dict, Optional, Union
```

Comment:
`Dict` can be nowadays type annotated as `dict[str, str]` which makes this legacy.